### PR TITLE
Add test usecase module dashproducts

### PR DESCRIPTION
**Pull request by matks**
This PR adds a new integration test case which is module dashproducts.

dashproducts contains a TPL file so this 2nd usecase helps to validate that header-stamp works well on Smarty files as well. I plan to add more usecase to cover different usecases where header-stamp behaves differently: Twig templates, Javascript files, CSS files.

This PR
- adds test/integration/module-samples/dashproducts folder
- adds test/integration/expected/dashproducts folder
- adapts run.php to enable the new module to validate
- adapts php-cs-fixer config to ignore module sample folders

### CODE REVIEW

File: tests/integration/runner/run.php [hunk 1]
```diff
@@ -11,15 +11,16 @@
...
+    'gsitemap',
+    'dashproducts'
 ];
 $workspaceID = 100;
 $filesystem = new Filesystem();
 $folderComparator = new FolderComparator();
 
 $application = buildTestApplication();
 
-foreach ($modulesToTest as $moduleName => $config) {
```
Comment (matks) [inline tests/integration/runner/run.php:22]: `$config` was an unused variable

File: .php_cs.dist [hunk 2]
```diff
@@ -6,6 +6,8 @@ $config
     ->setUsingCache(false)
     ->getFinder()
     ->in(__DIR__)
-    ->exclude('vendor');
+    ->exclude('vendor', 'tests/integration/module-samples', 'tests/integration/expected')
+    ->exclude('tests/integration/module-samples')
+    ->exclude('tests/integration/expected');
```
Comment (Quetzacoalt91) [inline .php_cs.dist:11]: Both are valid, but we do not need to exclude them two times :)

Comment (Quetzacoalt91) [inline .php_cs.dist:11]: `exclude()` handle an array of path or a single string. As your line will not be taken in account, we should have

```suggestion
    ->exclude(['vendor', 'tests/integration/module-samples', 'tests/integration/expected']);
```

or

```suggestion
    ->exclude('vendor')
    ->exclude('tests/integration/module-samples')
    ->exclude('tests/integration/expected');
```

Comment (matks) [reply .php_cs.dist]: I chose the 1st one https://github.com/PrestaShopCorp/header-stamp/pull/23/commits/559b92b59b8b97fb6327c3afc4524274cf1c5fed

**Comment by matks** [comment]
@Quetzacoalt91 Fixed !
Looks like I should stop pushing PRs at 10pm 🙄 the quality of my contribution is really not satisfying

Sorry for that

**Comment by Quetzacoalt91** [comment]
Thank you @matks !

**Pull request merged**